quarkus-extensions-with-dependency/src/main/java/io/quarkus/qe/DependencyFilter.java

```diff
@@ -0,0 +1,61 @@
+package io.quarkus.qe;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Objects;
+
+
+public class DependencyFilter {
+
+    public static void main(String[] args) throws IOException {
+        String currentWorkingDir = System.getProperty("user.dir");
+        System.out.println(currentWorkingDir);
+        File inputFile = new File(currentWorkingDir + "/quarkus-extensions.json");
```

Comment:
To handle possible exceptions working with files, it would be good to have a try(with resources) catch. As you did in ExcelUtils.
What do you think?

Reply:
`File` class is not closable. Work with the file is done later `JsonNode jsonNode = objectMapper.readTree(inputFile);`, that can throw `IOException` only